frb_codegen/src/generator/dart/mod.rs

```diff
@@ -338,7 +338,7 @@ fn has_methods(struct_name: String, ir_file: &IrFile) -> bool {
 
 // Tests if the function in `f` is a method for struct with name `struct_name`
 fn is_method_for_struct(f: &&IrFunc, struct_name: String) -> bool {
-    f.name.contains("__method")
+    f.name.contains(METHOD_MARKER)
```

Comment:
nit: maybe it is "endswith"?

Reply:
for `METHOD_MARKER=__method`, it can be, but for `STATIC_METHOD_MARKER=__static_method` don't because a static method is

`method_name__static_method___StructName`

I had to embed the struct for which the static method is. For common methods, it's always the first argument so no need

Reply:
Agree. So `contains` LGTM